app/controllers/admin_sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class AdminSessionsController < ApplicationController
+  def login
+    session = AdminSession.new(
```

Comment:
Should `admin_params` be passed here?

Reply:
What do you mean exactly?

Reply:
Oh nevermind, you are using `admin_params` here 👍